develop/docs/dlq.md

```diff
@@ -0,0 +1,28 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. Search `Task enqueued to DLQ` in logs and see task details.
+2. From the log trace, you can identify the offending namespace, task type, source cluster, and target cluster of the task.
```

Comment:
Mention that, if they can't find the logs, they can also just list the DLQs to see which ones are non-empty.